tensorboard/webapp/persistent_settings/_data_source/persistent_settings_data_source_test.ts

```diff
@@ -311,7 +316,7 @@ describe('persistent_settings data_source test', () => {
 
           expect(actual).toEqual({
             scalarSmoothing: 0.5,
-            tooltipSortString: 'default',
+            tooltipSortString: 'default' as TooltipSort,
```

Comment:
Use TooltipSort.DEFAULT?

Reply:
Because the test is testing deserialization from browser storage, I prefer testing the string literal.

Reply:
Very similar to my other response: The only serialized form of the string exists in LocalStorage. The contract of dataSource.getSettings() says it will return a TooltipSort object, so that seems to be the type you should use for the test.

I think the reasoning you are using would apply to setSettings() tests but it doesn't really apply to getSettings() tests.